src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  width: 100px;
+}
+
+.stars > :last-child {
+  margin-right: 4px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background: url(/images/star.svg) no-repeat center;
+}
+
+.stars--1 > :nth-child(-n+1),
```

Comment:
You can also use this option **.stars--1 .stars__star:nth-child(-n+1)**

Reply:
> You can also use this option **.stars--1 .stars__star:nth-child(-n+1)**

ok, i will use it